runtime/vm/exceptionsupport.c

```diff
@@ -940,6 +940,77 @@ setClassLoadingConstraintSignatureError(J9VMThread *currentThread, J9ClassLoader
 }
 
 
+void  
+setClassLoadingConstraintOverrideError(J9VMThread *currentThread, J9UTF8 *newClassNameUTF, J9ClassLoader *loader1, J9UTF8 *class1NameUTF, J9ClassLoader *loader2, J9UTF8 *class2NameUTF, J9UTF8 *exceptionClassNameUTF, U_8 *methodName, UDATA methodNameLength, U_8 *signature, UDATA signatureLength)
+{
+	PORT_ACCESS_FROM_VMC(currentThread);
+	char * msg = NULL;
+	const char * nlsMessage;
+
+	nlsMessage = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE, J9NLS_VM_LOADING_CONSTRAINT_OVERRIDE_VIOLATION, NULL);
+	if (nlsMessage != NULL) {
+		/* Loader1 name and length */
+		j9object_t loader1Object = loader1->classLoaderObject;
+		J9Class * loader1Class = J9OBJECT_CLAZZ(currentThread, loader1Object);
+		J9UTF8 * loader1ClassNameUTF = J9ROMCLASS_CLASSNAME(loader1Class->romClass);
+		U_16 loader1ClassNameLength = J9UTF8_LENGTH(loader1ClassNameUTF);
+		U_8 * loader1ClassName = J9UTF8_DATA(loader1ClassNameUTF);
+		I_32 loader1Hash = objectHashCode(currentThread->javaVM, loader1Object);
+
+		/* Loader2 name and length */
+		j9object_t loader2Object = loader2->classLoaderObject;
+		J9Class * loader2Class = J9OBJECT_CLAZZ(currentThread, loader2Object);
+		J9UTF8 * loader2ClassNameUTF = J9ROMCLASS_CLASSNAME(loader2Class->romClass);
+		U_16 loader2ClassNameLength = J9UTF8_LENGTH(loader2ClassNameUTF);
+		U_8 * loader2ClassName = J9UTF8_DATA(loader2ClassNameUTF);
+		I_32 loader2Hash = objectHashCode(currentThread->javaVM, loader2Object);
+
+		/* Class1 name and length */
+		U_16 class1ClassNameLength = J9UTF8_LENGTH(class1NameUTF);
+		U_8 * class1ClassName = J9UTF8_DATA(class1NameUTF);
+
+		/* Class2 name and length */
+		U_16 class2ClassNameLength = J9UTF8_LENGTH(class2NameUTF);
+		U_8 * class2ClassName = J9UTF8_DATA(class2NameUTF);
+
+		/* ExceptionClass name and length */
+		U_16 exceptionClassNameLength = J9UTF8_LENGTH(exceptionClassNameUTF);
+		U_8 * exceptionClassName = J9UTF8_DATA(exceptionClassNameUTF);
+
+		/* New class name and length */
+		U_16 newClassNameLength = J9UTF8_LENGTH(newClassNameUTF);
+		U_8 * newClassName = J9UTF8_DATA(newClassNameUTF);
+
+		UDATA msgLen;
```

Comment:
Nitpick: Initial value?

Reply:
This can just be merged with the line below to assign to it at creation

Reply:
Agreed - copied code again.

Reply:
Fixed all similar occurrences.